tools/mesh-generators/README.md

```diff
@@ -8,3 +8,20 @@ python generate_unit_cube.py --mesh  coarse.vtk --resolution 0.2
 ```
 
 This requires the `gmsh` and `meshio` Python packages.
+
+# Unit grid mesh generator
```

Comment:
The edge length of the mesh is always 1 in each dimension, right? Although this might be clear for you, would be good to add it in a sentence here.

Reply:
I can add it in the name, but "Unit" already means that :wink: